src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -6,20 +6,23 @@ import { DisplayMode, HealthIndicator } from '../../components/Health/HealthIndi
 import MissingSidecar from '../../components/MissingSidecar/MissingSidecar';
 import { PromisesRegistry } from '../../utils/CancelablePromises';
 import { ConfigIndicator } from '../../components/ConfigValidation/ConfigIndicator';
+import { ApiTypeIndicator } from '../../components/ApiDocumentation/ApiTypeIndicator';
 
 interface Props {
   item: ServiceListItem;
+  hasApiColumn: boolean;
 }
 interface State {
   health?: ServiceHealth;
+  columnWidth: number;
 }
 
 export default class ItemDescription extends React.PureComponent<Props, State> {
   private promises = new PromisesRegistry();
 
   constructor(props: Props) {
     super(props);
-    this.state = { health: undefined };
+    this.state = { health: undefined, columnWidth: props.hasApiColumn?3:4 };
```

Comment:
I don't really like the magic numbers here, specially without proper padding.

Reply:
I will change to nbColumns and make the computing of size in the display part